### PR TITLE
Scripts and input files for weak scaling on frontier

### PR DESCRIPTION
**Pull request by marchdf**

### CODE REVIEW

**Comment by jrood-nrel** [comment]
The yaml file might be confusing without the submit jobs script?

**Comment by marchdf** [comment]
Good point. I can remove or maybe point to job-scholar as the way to run it?

**Comment by jrood-nrel** [comment]
Sure, maybe just add a README.

File: weak-scaling/amrwind-frontier-weak-scaling-2023-11-28-17-32/amrwind-frontier-weak-scaling-1/amrwind-frontier-weak-scaling-1.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+cmd "module unload PrgEnv-cray"
+cmd "module load PrgEnv-amd"
+cmd "module load amd/5.4.3"
+cmd "export FI_MR_CACHE_MONITOR=memhooks"
+cmd "export FI_CXI_RX_MATCH_MODE=software"
+cmd "export MPICH_SMP_SINGLE_COPY_MODE=NONE"
+cmd "export SPACK_MANAGER=${HOME}/exawind/spack-manager"
+cmd "source ${SPACK_MANAGER}/start.sh && spack-start"
+cmd "spack env activate -d ${SPACK_MANAGER}/environments/amr-wind-mpi"
```
Comment (psakievich) [inline weak-scaling/amrwind-frontier-weak-scaling-2023-11-28-17-32/amrwind-frontier-weak-scaling-1/amrwind-frontier-weak-scaling-1.sh:28]: Would you add either the `spack.yaml` file for this environment, and/or directions on what flags were given for the constructing the environment?

**Review by psakievich — CHANGES_REQUESTED** [review]
Not sure I fully comprehend. Were all these subdirectories and the build generated by job-scholar? If so then disregard my comment on the env creation.  My main request is to add in a little more clarity for those who haven't used this job creation/submission tool before so they can reproduce these results with the minimal working set of instructions.  

Also if all these directories are auto created *and* job-scholar is considered stable then I don't think we need to commit them all.

Eventually it would be good if converged all these cases to use the same submission styles (right now there are at least 3), but I think it is appropriate to kick that can down the road right now.

**Review by psakievich — APPROVED** [review]

**Pull request merged**